packages/discord.js/src/managers/ThreadMemberManager.js

```diff
@@ -103,11 +103,13 @@ class ThreadMemberManager extends CachedManager {
 
   /**
    * Remove a user from the thread.
-   * @param {Snowflake|'@me'} id The id of the member to remove
+   * @param {UserResolvable|'@me'} member The member to remove
    * @param {string} [reason] The reason for removing this member from the thread
    * @returns {Promise<Snowflake>}
    */
-  async remove(id, reason) {
+  async remove(member, reason) {
+    const id = member === '@me' ? member : this.client.users.resolveId(member);
+    if (!id) throw new DiscordjsTypeError(ErrorCodes.InvalidType, 'member', 'UserResolvable');
```

Comment:
I am pretty sure introducing an error here where one was not before is a breaking change. If an `id` is `undefined`, one would handle the error from Discord, but all of a sudden it will no longer work as discord.js will be throwing a custom one.

Reply:
I disagree, this brings it in line with the `add` method

Reply:
Although versioning is a discussion for the project team, my own 2¢ is that whether this is breaking depends on which definition is used for the API standard: what's documented, or what's implemented.

Thrown error types aren't part of the API documentation, so there's nothing in the formal library specification that implies that this function _shouldn't_ throw a DiscordjsError if the function is called with the wrong signature (as per other library functions), nor that one should expect to be able to narrow-catch a DiscordAPIError if the function is called with the wrong signature, even if that's how the current implementation behaves.